tests/integration/new_relations/test_new_relations.py

```diff
@@ -569,3 +569,41 @@ async def test_invalid_extra_user_roles(ops_test: OpsTest):
             raise_on_blocked=False,
             timeout=1000,
         )
+
+
+async def test_indico_db_blocked(ops_test: OpsTest) -> None:
```

Comment:
We can now remove the `blocked` word from the test name.

Reply:
Updated.